debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -8,6 +8,7 @@
 import static io.debezium.connector.sqlserver.util.TestHelper.TYPE_LENGTH_PARAMETER_KEY;
```

Comment:
Could you also update `SqlServerConnectorIT#excludeColumnWhenCaptureInstanceExcludesColumns()` and remove the filter? As discussed in the Jira issue, this should not be needed any longer with your fix.